src/typecheck/reporting.rs

```diff
@@ -110,18 +107,73 @@ pub fn to_type(
     table: &UnifTable,
```

Comment:
I think the documentation comment isn't quite correct anymore. `var_to_type` and `cst_to_type` have been replaced by `var_name` and `cst_name` and generate the names directly, now.